packages/shell-api/src/field-level-encryption.ts

```diff
@@ -208,10 +215,34 @@ export class ClientEncryption extends ShellApiWithMongoClass {
     assertArgsDefinedType([keyId, value, options], [true, true, true], 'ClientEncryption.encryptExpression');
     return await this._libmongocrypt.encryptExpression(
       value,
-      // @ts-expect-error libmongocrypt typings contain a typo, see NODE-5023
       { keyId, ...options }
     );
   }
+
+  @returnsPromise
+  @apiVersions([])
+  async createEncryptedCollection(
+    dbName: string,
+    collName: string,
+    options: CreateEncryptedCollectionOptions
+  ): Promise<{ collection: Collection, encryptedFields: Document }> {
+    assertArgsDefinedType([dbName], ['string'], 'ClientEncryption.createEncryptedCollection');
+    assertArgsDefinedType([collName], ['string'], 'ClientEncryption.createEncryptedCollection');
+    assertArgsDefinedType([options], ['object'], 'ClientEncryption.createEncryptedCollection');
+    assertKeysDefined(options, ['provider', 'createCollectionOptions']);
+
+    const nativeClient = this._mongo._serviceProvider.getRawClient();
+    const db = nativeClient.db(dbName);
+    const { encryptedFields } = await this._libmongocrypt.createEncryptedCollection(
+      db,
+      collName,
+      options
+    );
+    return {
+      collection: new Collection(this._mongo, this._mongo.getDB(dbName), collName),
```

Comment:
Also how do we feel about this? I am returning our Collection object instead of native Collection object just to make sure users are able to chain mongosh Collection's methods on top of it.